package.json

```diff
@@ -32,7 +33,7 @@
   "dependencies": {
     "es6-collections": "0.x",
     "skatejs-named-slots": "1.x",
-    "webcomponents.js": "webcomponents/webcomponentsjs#b77ca74"
```

Comment:
Hmm I'm wondering why this worked. Doing a `git log | grep b77ca -B 10 -A 10` in skatejs/webcomponentsjs gives:

```bash
tshugart:webcomponentsjs tshugart$ git log | grep b77ca -B 10 -A 10
    References #595

commit d8c6b88b354872b999818205e011a2f32054762e
Author: Trey Shugart <treshugart@gmail.com>
Date:   Mon Aug 15 23:52:54 2016 +1000

    feat: add initial support for customised built-in elements.
    
    Implements #595

commit b77ca743edecb6ec262d4ffe8443708dbe0be49b
Author: Justin Fagnani <justinfagnani@google.com>
Date:   Tue Jun 28 15:18:02 2016 -0700

    Update spec references

commit 98b1d02a4bf6fb162d7d0be33468809f58147690
Author: Justin Fagnani <justinfagnani@google.com>
Date:   Sat Jun 25 09:27:39 2016 -0700

    Add instanceof HTMLElement support for built-in tags, mostly.
```

And doing that in webcomponentsjs doesn't uncover anything. It seems that the commit hash was from the skatejs fork, not webcomponentsjs. Interesting how this worked.